.gitignore

```diff
@@ -9,4 +9,5 @@ demos/paymentDirect/MangoPaySdkStorage.tmp.php
 /vendor/
 composer.lock
 
-.idea
\ No newline at end of file
+.phpunit.result.cache
+.idea
```

Comment:
This should not be added here but rather in your own local ignore file since it's not tied to the project but your environment.